app/src/components/trace/TokenCount.tsx

```diff
@@ -33,21 +32,69 @@ export function TokenCount(props: TokenCountProps) {
         <TokenItem size={props.size}>{props.tokenCountTotal}</TokenItem>
       </TriggerWrap>
       <Tooltip>
-        <Flex direction="column" gap="size-50">
-          <Flex direction="row" gap="size-100" justifyContent="space-between">
-            prompt tokens
-            <TokenItem>{props.tokenCountPrompt}</TokenItem>
-          </Flex>
-          <Flex direction="row" gap="size-100" justifyContent="space-between">
-            completion tokens
-            <TokenItem>{props.tokenCountCompletion}</TokenItem>
-          </Flex>
-        </Flex>
+        <TokenDetails nodeId={props.nodeId} />
```

Comment:
Wrap in suspense component, pass fallback={<Loading />

We can pair on a skeleton after